app/models/git_repository.rb

```diff
@@ -184,13 +187,16 @@ def instance_cache(key)
   # success: stdout as string
   # error: nil
   def capture_stdout(*command, dir: repo_cache_dir)
-    success, output = Samson::CommandExecutor.execute(
-      *command,
-      whitelist_env: ['HOME', 'PATH'],
-      timeout: 30.minutes,
-      err: '/dev/null',
-      dir: dir
-    )
-    output.strip if success
+    Tempfile.create('git-stderr') do |error_file|
+      success, output = Samson::CommandExecutor.execute(
```

Comment:
this needs a rebase ?

Reply:
@grosser This PR is targeting your `grosser/7` branch. Would you prefer it target `master`?

Reply:
🤔 I think the alternative option would be less confusing. Updated.